src/containers/Terms.jsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
```

Comment:
We removed similar language from our code base because we figured it was too organization specific.

Instead we have a config option: `PRIVACY_URL` (possibly badly named) which should be the link to any legal content for texters/etc.

Please remove. (feel free to change the name of PRIVACY_URL though)